manifest.md

```diff
@@ -0,0 +1,261 @@
+
+- `application/vnd.oci.image.manifest.v1+json`: Image manifest format
+- `application/vnd.oci.image.manifest.list.v1+json`: Manifest list, aka "fat manifest"
+- `application/vnd.oci.image.serialization.v1+json`: Container config JSON
+- `application/vnd.oci.image.rootfs.tar.gzip`: "Layer", as a gzipped tar
+
+## Manifest List
+
+The manifest list is the "fat manifest" which points to specific image manifests for one or more platforms.
+Its use is optional, and relatively few images will use one of these manifests.
+A client will distinguish a manifest list from an image manifest based on the Content-Type returned in the HTTP response.
+
+## *Manifest List* Field Descriptions
+
+- **`schemaVersion`** *int*
+
+  This field specifies the image manifest schema version as an integer. This
+  schema uses the version `2`.
+
+- **`mediaType`** *string*
+
+  The MIME type of the manifest list. This should be set to
+  `application/vnd.oci.image.manifest.list.v1+json`.
+
+- **`manifests`** *array*
+
+  The manifests field contains a list of manifests for specific platforms.
+
+  Fields of a object in the manifests list are:
+
+  - **`mediaType`** *string*
+
+    The MIME type of the referenced object. This will generally be
+    `application/vnd.oci.image.manifest.v1+json`, but it could also
+    be `application/vnd.oci.image.manifest.v1+json` if the manifest
+    list references a legacy schema-1 manifest.
+
+  - **`size`** *int*
+
+    The size in bytes of the object.
+    This field exists so that a client will have an expected size for the content before validating.
+    If the length of the retrieved content does not match the specified length, the content should not be trusted.
+
+  - **`digest`** *string*
+
+    The digest of the content, as defined by the [Registry V2 HTTP API Specificiation](https://docs.docker.com/registry/spec/api/#digest-parameter).
+
+  - **`platform`** *object*
+
+    The platform object describes the platform which the image in the manifest runs on.
+    A full list of valid operating system and architecture values are listed in the [Go language documentation for `$GOOS` and `$GOARCH`](https://golang.org/doc/install/source#environment)
+
+    - **`architecture`** *string*
+
+        The architecture field specifies the CPU architecture, for example `amd64` or `ppc64le`.
+
+    - **`os`** *string*
+
+        The os field specifies the operating system, for example `linux` or `windows`.
+
+    - **`os.version`** *string*
+
+        The optional os.version field specifies the operating system version, for example `10.0.10586`.
+
+    - **`os.features`** *array*
+
+        The optional os.features field specifies an array of strings, each listing a required OS feature (for example on Windows `win32k`).
+
+    - **`variant`** *string*
+
+        The optional variant field specifies a variant of the CPU, for example `armv6l` to specify a particular CPU variant of the ARM CPU.
```

Comment:
`This OPTIONAL property specifies the variant of the CPU, for example 'arm61' to specify a particular CPU variant of the ARM CPU.`